.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["airbnb"]
```

Comment:
let's add `add-module-exports` here, and also transform `Object.assign` - like https://github.com/airbnb/prop-types/blob/master/.babelrc#L4 for example

Reply:
sounds good, I'll add 'em!